.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,21 @@
+name: Pull Request
+on: pull_request
```

Comment:
![image](https://user-images.githubusercontent.com/13632762/83021552-8f217480-a000-11ea-8eec-6e8a337da727.png)

This change seems to not be working https://github.com/brainn-co/xcribe/actions/runs/116015650 @Danielwsx64

Reply:
Fixed! :v: